app/lib/metrics_service.rb

```diff
@@ -46,13 +46,15 @@ def self.get_default_headers(user)
   # This call links that anonId to the user's bearer token used by DSP's Sam
   # service.  That bearer token is in turn linked to a deidentified
   # "distinct ID" used to track users across auth states in Mixpanel.
-  def self.merge_identities_in_mixpanel(user)
+  def self.merge_identities_in_mixpanel(user, cookies)
```

Comment:
don't you need to update the corresponding call to this function in OmniauthCallbacksController with the new argument?  Or is there some rails auto-magic that makes that happen?

Reply:
Good catch.  I had it locally, but overlooked `add`ing the changed file.  Done in f9f3429.